File: foundation/foundation 3 month/lesson10/homework/ex_7.py
def check(flname):
	with open(flname,'wt') as w:
		k=input("Suzlarni kiritng: ").split()
		for x in k:
			w.write(x+' ')
		print("Kiritid")

def checks(flname):
	with open(flname,'rt') as r:
		dat=r.read().split(' ')
		dat.pop(-1)
		ms=list()
		for x in dat:
			ms.append(x)
		print(ms)
		arr=list
		for x in ms:
			cnt=0
			for y in x:
				if y in "AaUuOoEeIi":
					cnt+=1
			if cnt==0:
				print(x)


k=input("Fayl nomi: ")
check(k)
checks(k)

File: full stack/full stack 1 month/lesson6/homework/ex_7.js
let S = "example"
let k = 2
let s = 'k'
let arr = S.split('')
for(let i = 0; i< arr.length;i++)
{
    if (i%k == 0)
    {
    arr[i] = s
      
    }
}
S = arr.join("")
console.log(S)